src/main/groovy/com/github/spotbugs/snom/SpotBugsReport.java

```diff
@@ -50,8 +51,8 @@ public SpotBugsReport(ObjectFactory objects, SpotBugsTask task) {
   public abstract String toCommandLineOption();
 
   /** @deprecated use {@link #getOutputLocation()} instead. */
-  @Override
   @Deprecated
+  @ReplacedBy("getRequired")
```

Comment:
Let's just mark these as `@Internal`, they're treated as inputs elsewhere and `ReplacedBy` is meaningless outside of the Gradle codebase.